data/jsons/dependencies.json

```diff
@@ -227,7 +227,7 @@
 	    "headers": [
 		"<locale.h>"
 	    ],
-	    "fragment": "setlocale(LC_ALL, NULL);"
+	    "fragment": "freelocale(NULL);"
```

Comment:
hum, this is not the way to go. @glima need only `setlocale()`, and I guess other places will do as well.

Add a new check for `newlocale` and use `HAVE_NEWLOCALE` at the call site.